web/annotation_layer_builder.js

```diff
@@ -195,6 +209,23 @@ class AnnotationLayerBuilder {
     return !!this.annotationLayer?.hasEditableAnnotations();
   }
 
+  async injectLinkAnnotations(inferredLinks, viewport, structTreeLayer) {
+    const uniqueLinks = this.#annotations
```

Comment:
This is a somewhat strange/confusing name, since there's nothing "unique" about these links.
```suggestion
    const newLinks = this.#annotations
```

Reply:
Okay